source/dubregistry/dbcontroller.d

```diff
@@ -324,18 +341,10 @@ class DbController {
 		import std.math : round;
 
 		if (!query.strip.length) {
-			return m_packages.find()
-				.sort(["stats.score": 1])
-				.map!(deserializeBson!DbPackage)
-				.array;
+			return pkgCache;
 		}
 
-		auto pkgs = m_packages
-			.find(["$text": ["$search": query]], ["textScore": bson(["$meta": "textScore"])])
-			.sort(["textScore": bson(["$meta": "textScore"])]) // sort to only keep most relevant results
-			.limit(50) // limit irrelevant sort results (fixes #341)
```

Comment:
is the limit not needed anymore? does it crash if you search for example `e`?

Reply:
What's best, a limit on search length or amount?